File: src/praktikum2/latihan1/LinkedList.java
/**
 @author: rizkyhaksono
 */

package praktikum2.latihan1;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class LinkedList {

    ArrayList<String> namaHewan = new ArrayList<>(
            List.of("Sapi", "Kelinci", "Kambing", "Unta", "Domba"));

    public void hampusHewan(String del) {
        namaHewan.remove(del);
    }

    public void tambahHewan(String add) {
        namaHewan.add(add);
    }

    public void tampilHewan() {
        for (String hewan : namaHewan) {
            System.out.print(hewan + " ");
        }
    }

    public static void main(String[] args) {
        LinkedList linkedList = new LinkedList();
        linkedList.tampilHewan();

        try (Scanner scn = new Scanner(System.in)) {
            System.out.print("\nMasukkan nama hewan yang ditambah: ");
            String addHewan = scn.nextLine();

            linkedList.tambahHewan(addHewan);
            linkedList.tampilHewan();

            System.out.print("\nMasukkan nama hewan yang dihapus: ");
            String delHewan = scn.nextLine();

            linkedList.hampusHewan(delHewan);
        } catch (Exception e) {
            System.out.println(e);
        }
        linkedList.tampilHewan();
    }
}

File: src/praktikum2/latihan1/LinkedList.py
__author__      = "rizkyhaksono"
__copyright__   = "Copyright 2023, Malang"

class LinkedList:

    my_list = ["Sapi", "Kelinci", "Kambing", "Unta", "Domba"]
    print(my_list)

    # delete list item
    del my_list[0]
    print(my_list)

    # append list item
    my_list.append('Kuda')
    print(my_list)

    # insert list item
    my_list.insert(2, 'Rizky')
    print(my_list)